docs/virtual_machines/accessing_virtual_machines.md

```diff
@@ -92,6 +92,18 @@ ClusterRole using a RoleBinding that targets a single namespace.
 
 ## SSH Access
 
+In order to connect to a vm, you need to inject a key into it, and after the key is injected you can connect to it with "virtctl ssh".
+
+### Connecting to a vm with virtctl after injecting your key
+
+```
+virtctl ssh <user_name>@vm/<vm_name>.<namespace_name>
+```
+
+*Please not that "virtctl ssh" is only supported in up to date versions of virtctl.
```

Comment:
Please use an admonition here.  They make important note pop and grab the eye.

Example code : https://github.com/kubevirt/user-guide/blame/main/docs/virtual_machines/startup_scripts.md#L55-L56
Example render : http://kubevirt.io/user-guide/virtual_machines/startup_scripts/